### DISCUSSION TITLE
是否提供高精度模型的训练代码？

### DISCUSSION THREAD

**Discussion by tspp520** [PaddleCV-SIG/iann#61] [Q&A] [answered]
看了eiseg的readme，以及所有issues，好像在25天前作者提到会近期开源相关训练代码，
但readme上还是没有见到相关代码。
目前想基于工业界的某一细分领域的数据集训练一个模型去做 后续数据集的标注，以提高标注质量和速度，
是否提供 hrnet18_ocr64_cocolvis 这个高精度模型的训练代码？谢谢！

**Comment by geoyee** [comment, marked as answer]
感谢支持，不好意思，训练部分的话还需要一段时间。
